Auth/Auth.Platform.Android/AuthService.cs

```diff
@@ -74,7 +76,8 @@ public Task HandleAuthResult(Intent data)
         /// <param name="config">The service configuration. If <code>null</code> then <code>MobileCore.GetServiceConfiguration(Type)</code> is used.</param>
         public static IAuthService InitializeService(MobileCore core = null, ServiceConfiguration config = null)
         {
-            return MobileCore.Instance.RegisterService<IAuthService>(new AuthService(core, config));
+            ServiceFinder.RegisterInstance<ISecurityCheckFactory>(new AndroidSecurityCheckFactory(Android.App.Application.Context));
+            return (core ?? MobileCore.Instance).RegisterService<IAuthService>(new AuthService(core, config));
```

Comment:
What do you think about making core not null?

Reply:
The only reason we have the core parameter here is for testing purpose.
The user shouldn't really pass a core instance: MobileCore.Instance is usually used.

Reply:
In that case you can use named parameter:

```c#
var config = NotNullServiceConfiguration;
var AuthService = new AuthService(config: config);
```